packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -313,9 +313,12 @@ export const Webchat = forwardRef((props, ref) => {
           <Text
             id={input.id}
             from='user'
+            blob={false}
             style={{
               backgroundColor: COLORS.SCORPION_GRAY,
               borderColor: COLORS.SCORPION_GRAY,
+              padding: '10px 15px',
+              maxWidth: '54%',
```

Comment:
why are these two new properties needed?

Reply:
Because the message is no longer inside the blob, I could create a new prop inside the message to remove just the blobTick, but since blockInputs are not used very much I thought this could work.

Reply:
Consider to put here the same styles as the other messages (the ones from user/bot, I think it's `8px 12px`) in order to reduce the different variations of blobs (will be easy to maintain for us or to be refactored in a unified component in a future :) ). I think the width should not be necessary as if this component is flex, it should resize automatically depending on its inner content.